### PR TITLE
Upgrade to vibe.d 0.7.27.

### PR DESCRIPTION
**Pull request by s-ludwig**
The latest release of vibe.d contains considerable performance improvements. This updates the dependency specifications.

### CODE REVIEW

File: frameworks/D/vibed/dub.selections.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	"fileVersion": 1,
 	"versions": {
 		"memutils": "0.4.3",
```
Comment (Dith3r) [inline frameworks/D/vibed/dub.selections.json]: memutils: 0.4.4

File: frameworks/D/vibed/dub.selections.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
-		"vibe-d": "0.7.26",
+		"vibe-d": "0.7.27",
 		"libevent": "2.0.1+2.0.16",
 		"openssl": "1.1.4+1.0.1g",
 		"libasync": "0.7.5",
```
Comment (Dith3r) [inline frameworks/D/vibed/dub.selections.json]: "libasync": "0.7.8"

**Comment by Dith3r** [comment]
Expected to see 50% improvement over vibe.d 0.7.26.

**Comment by s-ludwig** [comment]
I'd expect even more than that, because 0.7.26 due to a performance regression barely scaled to multiple cores at all - certainly it wouldn't scale to a dual-Xeon-CPU setup with 24 cores.

**Comment by s-ludwig** [comment]
Updated the libasync/memutils dependencies (they are not used for the benchmark, but just in case).

**Comment by ssmith-techempower** [comment]
Looks good, merging in.

**Pull request merged**